src/styles/blocks/price.scss

```diff
@@ -0,0 +1,21 @@
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+
+  &__text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $thin-text-color;
+    margin: 0;
+  }
+
+  &__num {
+    font-weight: 700;
+    font-size: 16px;
```

Comment:
font-size: 16px; - default browser value, redundant

Reply:
Done! But to center button and image I had to use margin-right and margin-left. If it is not okay, please, let me know, I will fix it. Thank you for the feedback